src/libsync/discovery.cpp

```diff
@@ -572,7 +572,7 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
             // we need to make a request to the server to know that the original file is deleted on the server
             _pendingAsyncJobs++;
             auto job = new RequestEtagJob(_discoveryData->_account, originalPath, this);
-            connect(job, &RequestEtagJob::finishedWithResult, this, [=](const HttpResult<QString> &etag) mutable {
+            connect(job, &RequestEtagJob::finishedWithResult, this, [=, &path](const HttpResult<QString> &etag) {
```

Comment:
I don't think that's right since the job will be done async so the path variable will be a dangling pointer.